tests/testthat/helper-HTML.R

```diff
@@ -50,61 +50,18 @@ tests_init <- function(browserName = "phantomjs", dir = ".", port = 4848, ...) {
   if(OS == "Windows" || OS == "Darwin") {
     animint_server <- "host.docker.internal"
   }
-  if(browserName == "chromote"){
-    chrome.session <- chromote::ChromoteSession$new()
-    
-    chrome.session$view()
-    chrome.session$refresh <- function(){
-      ## from https://github.com/rstudio/chromote?tab=readme-ov-file#loading-a-page-reliably
-      prom <- chrome.session$Page$loadEventFired(wait_ = FALSE)  # Get the promise for the loadEventFired
```

Comment:
this code waits for the page to load, why are you deleting it?
did you forget to pull my new changes before you started to edit this file?

Reply:
@tdhock I kept everything inside chromote on my end. So I think it was due to not pulling your changes.

Reply:
@tdhock restored that code for page loading reliability. thanks